src/main/java/com/microsoft/azure/functions/worker/broker/EnhancedJavaMethodExecutorImpl.java

```diff
@@ -60,7 +60,7 @@ public void execute(BindingDataStore dataStore) throws Exception {
     }
 
     private Class<?> getContainingClass(String className) throws ClassNotFoundException {
-        return Class.forName(className, true, this.classLoader);
+        return Class.forName(className, false, this.classLoader);
```

Comment:
Another way to fix this issue is 
```
    private Class<?> getContainingClass(String className) throws ClassNotFoundException {
        try {
            Thread.currentThread().setContextClassLoader(this.classLoader);
            return Class.forName(className, false, this.classLoader);
        } finally {
            Thread.currentThread().setContextClassLoader(ClassLoader.getSystemClassLoader());
        }
    }
```

Currently cannot tell which is better. @trask appreciate it if you can provide some thoughts! Thanks so much！

Reply:
> some thoughts: first approach will delay the class initialization process to when the object is really created, which is during the function execution process and thus may impact the execution performance. Whereas, second approach will keep the class initialization at function load process, will not impact the execution performance

This is not accurate, as function load request also counts into cold start time in first request. Both approach would have same performance.